db/seeds.rb

```diff
@@ -5,6 +5,8 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+Project.create! id: 8, display_name: 'Oceans Eight'
+Project.create! id: 2501, display_name: 'Ghost in The Shell'
```

Comment:
Not necessarily. Since they're independent reducibles, they don't _need_ to be associated.

Reply:
Then we have reductions that refer to a reducible that doesn't exist? Because of the polymorphism we can't set up a foreign key relation on it, so PG won't complain, but I don't think the code is set up for this case.